chemie/web_push/models.py

```diff
@@ -0,0 +1,55 @@
+from django.db import models
+from push_notifications.models import APNSDevice, GCMDevice
+import datetime
+from django.db.models.signals import pre_delete
+from django.contrib.auth.models import User
+
+HC_ICON = "https://chemie.no/static/favicons/android-chrome-192x192.png"
+
+
+class CoffeeSubmission(models.Model):
+    """ Submission thats created each time a notification is send """
+    date = models.DateTimeField(default=datetime.datetime.now)
+
+    @classmethod
+    def check_last_submission(cls):
```

Comment:
Django ORM gjør veldig mye av dette for oss ut av boksen. 
Spesielt `timezone` og `timedelta` er veldig hendige. Da kunne det blitt noe ala:
```
15_minutes = datetime.timedelta(0, 900)
return cls.objects.filter(date__gte=timezone.now()-15_minutes).exists()
```

Reply:
Det var ganske smud løsning =)

Reply:
Funker utcnow()? Trodde umiddelbart at dette var en greie for utc og ikke utc+1 der vi er? Pleier å bare bruke now(). Hva er forskjellen? Kan også bruke .exists() i stedet for == 0 i starten